### DISCUSSION TITLE
Accessories handling

### DISCUSSION THREAD

**Discussion by aj-bi** [snipe/snipe-it#13529] [Ideas]
I would like to have the option to enter individual orders for the accessories.

Instead of one quantity field, it should be possible to insert several quantity fields.

Per added quantity field, it should then also be possible to upload the associated invoice.

Furthermore, it should be possible to delete a single accessory. This should then be subtracted from the quantity.

What do you think?

**Comment by noahammond** [comment]
This would be ideal for bulk orders. At the moment I'm having to keep a list of orders in the notes tab that look like this:
23/08/23 - Added Order ([QUOTE NO]) of 5
13/09/23 - Added Order ([QUOTE NO]) of 6

It works for the time being but I would love to bulk add a number of accessories with the option of adding a order number / note to the added amount.
